tools/unix/generate_localizations.sh

```diff
@@ -58,11 +58,10 @@ STRINGS_PATH="$OMIM_PATH/data/strings"
 MERGED_FILE="$(mktemp)"
 cat "$STRINGS_PATH"/{strings,types_strings}.txt> "$MERGED_FILE"
 
-# TODO: Add "--untagged --tags android" when tags are properly set.
 # TODO: Add validate-strings-file call to check for duplicates (and avoid Android build errors) when tags are properly set.
```

Comment:
can't use twine validation still as it doesn't allow dots `.` in definition names (all our types use dots e.g. like `[type.highway.footway]`

Reply:
Let's patch our twine fork :)

Reply:
Ok, maybe I'll do it sometime. Its not really important as our `strings_utils.py` does a good validation job already.